lib/_stream_wrap.js

```diff
@@ -1,3 +1,5 @@
 'use strict';
 
 module.exports = require('internal/js_stream_socket');
+process.emitWarning('The _stream_wrap module is deprecated.',
+                    'DeprecationWarning', 'DEP0124');
```

Comment:
The concrete deprecation number should not be used right now. Instead, just use `DEP0XXX`. It will be detected by our tools that this has to be replaced. The same applies to all other occurrences.

Reply:
The tools only detect `DEP00XX ` AFAIK. Anyway, it should be replaced when the PR is landed

Reply:
@targos I fixed that a while ago. The check is: `grep -q DEP...X doc/api/deprecations.md`. So the only important thing is that the last character is `X`.